package.json

```diff
@@ -64,7 +64,7 @@
     "react-dom": "18.2.0",
     "tsm": "^2.2.2",
     "tsup": "^6.2.3",
-    "typescript": "^4.9.5",
+    "typescript": "4.9.5",
```

Comment:
Do we want to pin TypeScript version only or other dependencies as well?

Reply:
I believe we should pin everything. I will do that in next PRs - it will require additional testing to ensure everything stays in place